docs/public-networks/how-to/upgrade-node.md

```diff
@@ -2,32 +2,240 @@
+metadata:
+  name: besu-data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Ti
+```
+
+## Upgrade on Ansible
+
+You can use the [Ansible role on Galaxy](https://galaxy.ansible.com/ui/standalone/roles/consensys/hyperledger_besu/)
+directly or customize it to suit your needs.
+
+Upgrade the Besu version on nodes by running the play with the new version.
+For more information, select **Documentation** on the [Ansible Galaxy Besu page](https://galaxy.ansible.com/ui/standalone/roles/consensys/hyperledger_besu/).
 
 The playbook:
 
 1. Stops Besu.
-1. Downloads the updated version.
-1. Applies any new configuration.
-1. Starts Besu.
+2. Downloads the updated version.
+3. Applies any new configuration.
+4. Starts Besu.
+
+## Verify post-upgrade
+
+### RPC methods
+
+If you have [JSON-RPC HTTP enabled](../reference/cli/options.md#rpc-http-enabled),
+you can use the following commands to verify that you've successfully upgraded your Besu node.
+
+Call [`eth_syncing`](../reference/api/index.md#eth_syncing) to check the node synchronization status:
+
+ ```bash
+ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":1}' http://127.0.0.1:8545
+ ```
+
+Call [`web3_clientVersion`](../reference/api/index.md#web3_clientversion) to check the current client version:
+
+```bash
+curl -X POST --data '{"jsonrpc":"2.0","method":"web3_clientVersion","params":[],"id":1}' http://127.0.0.1:8545
+```
+
+Call [`net_peerCount`](../reference/api/index.md#net_peercount) to verify peer connections:
+
+ ```bash
+ curl -X POST --data '{"jsonrpc":"2.0","method":"net_peerCount","params":[],"id":1}' http://127.0.0.1:8545
+ ```
+
+### Logs
+
+You can also check Besu's logs to verify the version and whether Besu is in sync.
+For example, the startup logs look like the following:
+
+```bash
+{"@timestamp":"2025-01-17T07:23:03,791","level":"INFO","thread":"main","class":"Besu","message":"Starting Besu","throwable":""}
+{"@timestamp":"2025-01-17T07:23:04,558","level":"INFO","thread":"main","class":"Besu","message":"Connecting to 0 static nodes.","throwable":""}
+{"@timestamp":"2025-01-17T07:23:04,643","level":"INFO","thread":"main","class":"Besu","message":"
+####################################################################################################
+#                                                                                                  #
+# Besu version 25.1.0                                                                #
+#   
+...                                                                                               #
+```
+
+### Metrics
+
+If you have [metrics enabled](../reference/cli/options.md#metrics-enabled), you can verify the version by checking the
+`process_release` metric in [Prometheus](monitor/metrics.md), or on the command line:
+
+```bash
+curl -s localhost:9545/metrics | grep process_release
+process_release{version="besu/v25.10/linux-x86_64/openjdk-java-21"} 1.0
```

Comment:
For the other curl commands we don't show the response, but we do here - do you think we should make this consistent one way or the other?

I think here at least we should separate the response to make it clear "process_release{version..." isn't part of the command to run.